nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/NiFiServiceFacade.java

```diff
@@ -2674,7 +2683,7 @@ ControllerServiceReferencingComponentsEntity updateControllerServiceReferencingC
     /**
      * @return the system diagnostics
      */
-    SystemDiagnosticsDTO getSystemDiagnostics();
+    SystemDiagnosticsDTO getSystemDiagnostics(boolean verbose);
```

Comment:
Should we make this argument an enum named `DiagnosticLevel` or something? Perhaps this won't change much, but it might be worth considering.

Reply:
I like it.